TestsFoundation/TestsFoundation/Helpers/GradesHelper.swift

```diff
@@ -28,9 +28,10 @@ public class GradesHelper: BaseHelper {
     public static func gradeOutOf(assignment: DSAssignment? = nil,
                                   assignmentId: String? = nil,
                                   actualPoints: String,
-                                  maxPoints: String) -> XCUIElement {
+                                  maxPoints: String, letterGrade: String = "") -> XCUIElement {
```

Comment:
Not something significant but other parameters are all on a new line, let's follow the patter with this one.